database/Group5Database.py

```diff
@@ -48,6 +49,16 @@ def get_graph(self):
             'messages': messages
         }
 
+    def get_messages(self):
+        result = self.get_graph()
+        return result['messages']
```

Comment:
Hier könnte man es vielleicht eher umgekehrt lösen: get_mssages verwaltet die Query für die messages und get_graph verwendet dann get_messages. Aktuell wird in get_messages eine überflüssige Query für die Personen abgesetzt, oder?

Reply:
Ja du hast Recht und ich hatte auch versucht das so umzusetzen, damit eben nicht die Query für die Personen überflüssig abgesetzt wird. Problem ist nur, dass die messages bei der Datenbank ja in Abhängigkeit von den Fraktionen ermittelt werden (weil sie halt keine Nodes sind) und das sentiment nicht als Eigenschaft gespeichert wird, sondern mit einer extra Query berechnet wird. Das hat mich aus der Bahn geworfen, kein Plan wie man das jetzt macht  . . .

Reply:
oh, du hast vollkommen Recht! Sorry, hatte ich nicht im Blick. Dann lass es gerne erstmal so.